contracts/LockedNORI.sol

```diff
@@ -734,4 +744,30 @@ contract LockedNORI is
     }
     _burn(account, amount);
   }
+
+  /**
+   * @dev See {IERC777-operatorBurn}. Reverts, unsupported.
+   *
+   * Emits {Burned} and {IERC20-Transfer} events.
+   */
+  function operatorBurn(
+    address,
+    uint256,
+    bytes memory,
+    bytes memory
+  ) public virtual override {
+    revert();
+  }
+
+  /**
+   * @dev See {IERC777-revokeOperator}.
+   */
+  function revokeOperator(address operator)
```

Comment:
this seems to definitely be missing in the base though, so might be the only thing to merge from this branch